docs/manual/installation/contao-manager.de.md

```diff
@@ -97,6 +97,10 @@ Der Contao Manager benötigt keine eigene Datenbank. Die Konfiguration des Conta
 <code>manager.json</code> und die Benutzerdaten in der <code>users.json</code> im Verzeichnis 
 <code>/contao-manager</code> gespeichert.
 
+{{% notice warning %}}
```

Comment:
Dieser Teil trifft doch für jede Software zu, ich würde die Notice in beiden Sprachen löschen. Ansonsten müssten wir beim Install-Tool auch darauf hinweisen.

Reply:
Der CM ist schnell kopiert. Ohne weitere Vorgehensweise (Konfiguration oder umbenennen) wäre es halt unsicher.
Aber wie du meinst. 
Soll ich die entfernen?

Reply:
Wenn ein System ohne spezifisches Wissen kopiert wird, ist es in meinen Augen immer unsicher. Gerne entfernen. 
Dann lieber eine Liste: was musst du beim Livegang deiner Website beachten.

Reply:
OK. Done.